.github/workflows/docs.sh

```diff
@@ -23,13 +23,13 @@ java -version
 javadoc -J-version
 
 # setup hugo
-HUGO_REPO=https://github.com/gohugoio/hugo/releases/download/v0.80.0/hugo_extended_0.80.0_Linux-64bit.tar.gz
-HUGO_ARTIFACT=hugo_extended_0.80.0_Linux-64bit.tar.gz
+HUGO_REPO=https://github.com/gohugoio/hugo/releases/download/v0.98.0/hugo_extended_0.98.0_Linux-64bit.tar.gz
+HUGO_ARTIFACT=hugo_extended_0.98.0_Linux-64bit.tar.gz
 if ! curl --fail -OL $HUGO_REPO ; then
 	echo "Failed to download Hugo binary"
 	exit 1
 fi
-tar -zxvf $HUGO_ARTIFACT
+tar -zxvf $HUGO_ARTIFACT -C /usr/local/bin
```

Comment:
Don't the `./hugo` calls below need to updated?

Reply:
Yes of course :(